src/main/java/com/sparta/vicky/jwt/jwtUtil.java

```diff
@@ -0,0 +1,108 @@
+package com.sparta.vicky.jwt;
+
+import io.jsonwebtoken.*;
+import io.jsonwebtoken.security.Keys;
+import jakarta.annotation.PostConstruct;
+import jakarta.servlet.http.HttpServletRequest;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+
+import java.security.Key;
+import java.util.Base64;
+import java.util.Date;
+
+@Slf4j
+@Component
+public class jwtUtil {
```

Comment:
자바에서 Util 클래스는 보통 final 클래스로, static 메서드를 제공하는 클래스입니다. 그래서 튜터님도 JwtUtil 보단, JwtProvider 같은 이름을 추천해주셨습니다.
```suggestion
public class jwtProvider {
```